src/mode.js

```diff
@@ -76,7 +76,7 @@ function getMode_(win) {
     // amp-geo override
     geoOverride: hashQuery['amp-geo'],
     test: coreMode.isTest(win),
-    log: hashQuery['log'],
+    log: parseInt(hashQuery['log'], 10),
```

Comment:
nit: `mode.log` is only used by `src/log.js`. can leave it here for this PR, but IMO this should be moved to `src/log.js`

Reply:
Agreed. `Log` and `mode` are both so needlessly complex and have pulled in so many things they shouldn't, every little bit we can pick apart is helpful. Will do this in a follow-up